docker-compose.yml

```diff
@@ -10,10 +10,8 @@ services:
     <<: &service_defaults
       image: ynr:test
       env_file:
-        - path: env/frontend.env
+        - path: .env
```

Comment:
`env/frontend.env` was badly named because we use it for both the web and worker containers
I've moved this to just `.env`
I don't think we're going to have a case for multiple env files any time soon. If we do, lets cross that bridge if we come to it.